raiden/network/transport/matrix/transport.py

```diff
@@ -503,6 +508,10 @@ def start_health_check(self, node_address):
             self.whitelist(node_address)
             self._address_mgr.add_userids_for_address(node_address, user_ids)
 
+            self.storage.write_matrix_userids_for_address(
```

Comment:
Not sure why mypy is ok with this

Reply:
How about introducing a new "storage" wrapper around sqlite3 that is only matrix-specific? i feel like adding everything altogether could be a maintaince nightmare at some point.

Reply:
I don't think it's necessary for two isolated tables. Also, we have some data stored which is needed by both client and transport, e.g. auth-data, and the set of healtchecked nodes

Reply:
> I don't think it's necessary for two isolated tables.

Not sure if i understand what you mean or if you didn't understand my meaning. Right now, we have a (Serialized)SQLiteStorage class which is meant to be a sqlite wrapper for the state machine. Adding matrix-related methods there would lead alot of intermingled code.

I really think you should move all the functionality you added to a MatrixStorage class